js/customscripts.js

```diff
@@ -736,5 +769,32 @@ function initToc() {
         }
     }
 
+    function checkHash() {
+        if (location.hash) {
+            let target = $(location.hash);
+
+            $window.scrollTop(target.offset().top - pageOffset + 1);
+        }
+    }
+
+    checkHash();
+
+    $window.on('load', function () {
+        $body.trigger('scroll');
+    });
+
     $window.on('resize orientationchange', updateOffset);
+
+    $window.on('activate.bs.scrollspy', function (e) {
+        let activeLink = tocLinks.filter(function(i, item){
```

Comment:
It looks like `find` is better fits here. If you are using filter then you have an array at the end, but you just need one element, so suggest to replace `filter` with `find`.